packages/davinci-client/src/lib/node.slice.ts

```diff
@@ -274,24 +273,19 @@ export const nodeSlice = createSlice({
       return state.client;
     },
     selectCollectors: (state) => {
-      // Let's check if the node has a client and collectors
-      if (state.status !== CONTINUE_STATUS) {
-        console.error(
-          `\`collectors\` are only available on nodes with \`status\` of ${CONTINUE_STATUS}`,
-        );
-        return [];
+      if (state.client && 'collectors' in state.client) {
+        return state?.client.collectors;
       }
-      return state.client.collectors || [];
+      return [];
     },
     selectCollector: (state, id: string) => {
-      // Let's check if the node has a client and collectors
-      if (state.status !== CONTINUE_STATUS) {
-        console.error(
-          `\`collectors\` are only available on nodes with \`status\` of ${CONTINUE_STATUS}`,
-        );
-        return;
+      if (state.client && 'collectors' in state.client) {
+        return state.client.collectors?.find((collector) => collector.id === id);
       }
-      return state.client.collectors?.find((collector) => collector.id === id);
+      console.error(
```

Comment:
It seems we preserved the error log for this selector, but the plural selector we removed it. We should try to stay consistent.